cinema/serializers.py

```diff
@@ -1,24 +1,85 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Genre, Actor, CinemaHall
+
+
+class GenreSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    name = serializers.CharField(required=True)
+
+    class Meta:
+        model = Genre
+
+    def create(self, validated_data: dict) -> Genre:
+        return Genre.objects.create(**validated_data)
+
+    def update(self, instance: Genre, validated_data: dict) -> Genre:
```

Comment:
use ModelSerializer
create and update methods are included

Reply:
OK, I need time to check next lessons first to know about ModelSerializer. I will be back once I watch all the new videos